meson.build

```diff
@@ -0,0 +1,31 @@
+#  Copyright © 2022 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+project('lc3', 'c',
+	version: '0.1',
+	license: 'Apache-2.0',
+	meson_version: '>= 0.47.0',
+	default_options: ['b_lto=true'])
+
+add_project_arguments('-ffast-math', language: 'c')
```

Comment:
Since this flag is just here for a free optimization, IMO it makes sense to do it like this:

```meson
add_project_arguments(cc.get_supported_arguments('-ffast-math'), language: 'c')
```

This only adds the option if the compiler accepts the option, which GCC does but e.g. MSVC does not. Admittedly it's not clear to me that the project is tested or expected to compile on Windows/MSVC.

Reply:
> @kirankrishnappa-intel Can you add this patch ? I will merge just after. Thanks.

@asoulier  - Yes... happy to amend the patch. Got busy with some other work. I will publish the changes tomorrow. Thanks.

Reply:
@asoulier - Looks  some compilation issue in Windows. If you think there is no need to support Windows as of now, we can go ahead merging the current patch.

> C:\Users\kirank1\Downloads\liblc3>meson compile -C build
> Activating VS 17.2.4
> ninja: Entering directory `C:/Users/kirank1/Downloads/liblc3/build'
> [1/13] Compiling C object src/lc3.dll.p/plc.c.obj
> FAILED: src/lc3.dll.p/plc.c.obj
> "cl" "-Isrc\lc3.dll.p" "-Isrc" "-I..\src" "-I..\include" "/MDd" "/nologo" "/showIncludes" "/utf-8" "/W2" "/Od" "/Zi" "/Fdsrc\lc3.dll.p\plc.c.pdb" /Fosrc/lc3.dll.p/plc.c.obj "/c" ../src/plc.c
> C:\Users\kirank1\Downloads\liblc3\include\lc3.h(186): error C2229: struct '<unnamed-tag>' has an illegal zero-sized array
> C:\Users\kirank1\Downloads\liblc3\include\lc3.h(187): error C2229: struct '<unnamed-tag>' has an illegal zero-sized array
> C:\Users\kirank1\Downloads\liblc3\include\lc3.h(189): error C2229: struct '<unnamed-tag>' has an illegal zero-sized array
> C:\Users\kirank1\Downloads\liblc3\include\lc3.h(190): error C2229: struct '<unnamed-tag>' has an illegal zero-sized array
> C:\Users\kirank1\Downloads\liblc3\src\fastmath.h(41): warning C4305: 'initializing': truncation from 'double' to 'const float'
> C:\Users\kirank1\Downloads\liblc3\src\fastmath.h(73): warning C4305: 'initializing': truncation from 'double' to 'const float'
> C:\Users\kirank1\Downloads\liblc3\src\common.h(29): fatal error C1083: Cannot open include file: 'stdalign.h': No such file or directory
> [2/13] Compiling C object src/lc3.dll.p/lc3.c.obj
> 
>

Reply:
@kirankrishnappa-intel The zero-sized array (or flexible array) is C99, and stdalign  C11. Perhaps miss a flag to specify the C standard revision. About the warnings, they are weird... Can be largely ignored.
By the way, support of windows is not really relevant as of now.
So Ok to merge, thanks.